lib/metrics.js

```diff
@@ -92,6 +93,24 @@ function openwhisk(metrics) {
     return metrics;
 }
 
+/**
+ * Container memory size information from the environment
+ *
+ * @param {Object} [metrics={}] Metrics object
+ * @returns {Object} Metrics object
+ */
+function containerMemory(metrics) {
+    metrics = metrics || {};
+    try {
+        metrics.containerMemorySize = parseFloat(fs.readFileSync('/sys/fs/cgroup/memory/memory.limit_in_bytes').toString()) || undefined;
```

Comment:
Since it is in bytes, would parseInt not be more adequate? I mean we can't have decimal values here - or did you choose parseFloat because of the need to handle large values?

Reply:
I used `parseFloat` because that is what we use in [cgroup-metrics](https://github.com/adobe/node-cgroup-metrics/blob/master/lib/utils.js#L55)

I am fine with switching to `parseInt`

Reply:
I see. As you wish. Btw, you don't need to trim a string value given to parseFloat, whitespaces will be ignored (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat, description).

Reply:
(I would move to parseInt to enforce the decimal radix, parseInt(value, 10)).